tests/foreman/api/test_classparameters.py

```diff
@@ -992,41 +1354,20 @@ def test_positive_remove_matcher(self):
         2. Remove the matcher created in step 1.
 
         @assert: The matcher removed from parameter.
-
-        @caseautomation: notautomated
-
-        @CaseLevel: Integration
-        """
-
-    @run_only_on('sat')
-    @stubbed()
-    @tier3
-    def test_positive_impact_parameter_delete_attribute(self):
```

Comment:
Why this stub is been deleted its valid stub, I discuss with @oshtaier as well you can get help from him in my absence.